src/pypi.rs

```diff
@@ -0,0 +1,126 @@
+//! Warehouse PyPI API Implementation
+
+use log::info;
+use serde::{Deserialize, Serialize};
+use std::fmt::Display;
+
+/// Download stats
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+pub struct PyPIPackageDownloadInfo {
```

Comment:
```suggestion
pub struct PypiPackageDownloadInfo {
```
The typical Rust convention is to keep acronyms with the first letter in uppercase, and the rest in lowercase. This would also need to be updated in the rest of the file.

Reply:
John wanted this formatting and I think matching the Offical naming looks better.